lib/generators/rest_api_generator/helpers.rb

```diff
@@ -17,5 +17,14 @@ def editable_attributes
         Rails::Generators::GeneratedAttribute.new(column.name.to_s, column.type.to_s)
       end
     end
+
+    def versioning?
+      !options["scope"].empty?
+    end
+
+    def version
+      parts = options["scope"].split(".")
+      "module " + parts[0].capitalize + '::' + parts[1].capitalize
+    end
```

Comment:
This works for only two version? Can we change this to work with n versions?

Reply:
done, now it works with n versions